data-raw/rs_supprs_onco_imwg.R

```diff
@@ -0,0 +1,123 @@
+# Packages: ---------------------------------------------------------------
```

Comment:
Please can you use the same header style as for other programs in `data_raw`? Below I include the one for `ae` as an example:
```
# Datasets: ae, suppae
# Description: Standard AE, SUPPAE datasets from CDISC pilot study

# Load libraries -----
library(dplyr)
library(metatools)
library(haven)
library(admiral)

# Create ae, suppae ----
...
```

Reply:
Updated